pkg/util/test/util.go

```diff
@@ -121,10 +121,10 @@ const (
 var TestSpecPath = "testspec"
 
 var (
-	pxVer2_12, _   = version.NewVersion("2.12.0-")
-	opVer1_10, _   = version.NewVersion("1.10.0-")
-	opVer1_9_1, _  = version.NewVersion("1.9.1-")
-	opVer1_10_1, _ = version.NewVersion("1.10.1-")
```

Comment:
are we skipping op 1.10.1 and plan to have this in 1.10.2? that case we still need 1.10.2- to handle 1.10.2-dev

Reply:
It was never the plan to deliver this change in 1.10.1. Earlier change was, if operator version <= 1.10.1, use old validation logic. This was however failing when tested for Operator 1.10.1 because of the comparison against the reg-ex 1.10.1- . So I decided to do the validation against 1.10.2 and below.

opVersion, _ := version.NewVersion("1.10.1")
opVersion_1_10_1, _ := version.NewVersion("1.10.1-")
opVersion_1_10_1_dev, _ := version.NewVersion("1.10.1-dev")

logrus.Warnf("opVersion.LessThan(opVersion_1_10_1) = %v", opVersion.LessThan(opVersion_1_10_1))
logrus.Warnf("opVersion.LessThanOrEqual(opVersion_1_10_1) = %v", opVersion.LessThanOrEqual(opVersion_1_10_1))
logrus.Warnf("opVersion.LessThan(opVersion_1_10_2_dev) = %v", opVersion.LessThan(opVersion_1_10_1_dev))
logrus.Warnf("opVersion.LessThanOrEqual(opVersion_1_10_2_dev) = %v", opVersion.LessThanOrEqual(opVersion_1_10_1_dev))
-->
time="2022-12-19T16:31:24-07:00" level=warning msg="opVersion.LessThan(opVersion_1_10_1) = false"
time="2022-12-19T16:31:24-07:00" level=warning msg="opVersion.LessThanOrEqual(opVersion_1_10_1) = false"
time="2022-12-19T16:31:24-07:00" level=warning msg="opVersion.LessThan(opVersion_1_10_2_dev) = false"
time="2022-12-19T16:31:24-07:00" level=warning msg="opVersion.LessThanOrEqual(opVersion_1_10_2_dev) = false"